### DISCUSSION TITLE
Quick command sorting problem

### DISCUSSION THREAD

**Discussion by ping-kong** [electerm/electerm#3167] [Ideas]
能不能在新建快速命令时添加序号，方便快速命令的排序?谢谢!
现在的快速命令排序规律像开盲盒~
![1](https://github.com/electerm/electerm/assets/48009676/def59fe5-566b-4ab5-8af3-90777c10c0de)
![2](https://github.com/electerm/electerm/assets/48009676/8e761194-6358-4299-b5fe-690362786b92)

**Comment by zxdong262** [comment]
将在后续版本加入，当前是按照使用次数排序

**Comment by zxdong262** [comment]
最新版已经可以拖拽更改顺序

**Comment by ping-kong** [comment]
谢谢，很好用
